src/Tribe/Editor/Blocks/Rsvp.php

```diff
@@ -375,8 +378,7 @@ public function rsvp_form() {
 	 * @return void
 	 */
 	public function rsvp_process() {
-
-		$response  = array( 'html' => '', 'view' => 'rsvp-process' );
+		$response  = [ 'html' => '', 'view' => 'rsvp-process' ];
```

Comment:
:no_entry_sign: When a multi-item array uses associative keys, each value should start on a new line.

<sub>hash: b7959952ec3a44178616958f071e60ea</sub>